modules/openapi/test/resources/foo.smithy

```diff
@@ -29,7 +29,10 @@ structure Person {
   name: String,
 
   @httpHeader("X-Bamtech-Partner")
-  partner: String
+  partner: String,
+
```

Comment:
It'd be good to add test cases for query params, path params, and body as well. Just to make sure that all the dates are ending up in the format they should

Reply:
@lewisjkl sure